src/dependency-manager/src/service-config/v1.ts

```diff
@@ -93,6 +99,10 @@ export class ServiceConfigV1 extends ServiceConfig {
     return this.api;
   }
 
+  getInterfaces(): { [name: string]: ServiceInterfaceSpec } {
+    return this.interfaces || {};
```

Comment:
```ts
const default_port = this.port || 8080;
return this.interfaces || { default_port: { port: default_port } };
```
Thoughts on always having an interface? Then we don't need any conditional logic elsewhere.